src/vscode-atopile/package.json

```diff
@@ -126,6 +127,48 @@
                 "title": "Restart Server",
                 "category": "atopile",
                 "command": "atopile.restart"
+            },
+            {
+                "command": "atopile.create",
+                "title": "Ato Create",
```

Comment:
The 'atopile' prefix is on all the commands so you can just type atopile to see all the commands, otherwise, they would be mixed in with all the rest of the available commands. I believe that is the most commonly expected behavior for VS Code addons.

Reply:
Yep! That bit make sense.

I meant the "Ato" on "Ato create". Wouldn't the pattern just be "create"

<img width="431" alt="Screenshot 2024-03-08 at 14 31 55" src="https://github.com/atopile/atopile/assets/4241578/b8d07d11-0874-4000-a35f-cf3152747abe">